code/modules/clothing/head/_head.dm

```diff
@@ -1,11 +1,13 @@
 /obj/item/clothing/head
-	name = "head"
-	icon_state = ICON_STATE_WORLD
-	icon = 'icons/clothing/head/softcap.dmi'
-	body_parts_covered = SLOT_HEAD
-	slot_flags = SLOT_HEAD
-	w_class = ITEM_SIZE_SMALL
-	blood_overlay_type = "helmetblood"
+	name                = "head"
+	icon_state          = ICON_STATE_WORLD
+	icon                = 'icons/clothing/head/softcap.dmi'
+	blood_overlay_type  = "helmetblood"
+	w_class             = ITEM_SIZE_SMALL
+	flags_inv           = BLOCKHEADHAIR
```

Comment:
This should not be the default behavior for all head items.

Reply:
The flag_inv you mean? Because the rest is just copy-pasted.

Reply:
Correct.

Reply:
Can you explain why though? Because 90% of all headgear hides only the hair, and not the facial hair? It logically avoids having to type ``flags_inv = BLOCKHEADHAIR`` for most of them?

Reply:
The basic hat behavior should not be to hide hair, the majority of hats don't. Helmets and space helmets can have the hair hiding flags set easily.

Reply:
Ok, misunderstood the flags.